File: sentimental-mario-less/mario.py
from cs50 import get_int
while True:
    height= get_int("Height:")
    if height >=1 and height<=8:
        break

for i in range(1,height+1):
    leer= height-i
    hash= i
    print(leer*" ", end="")
    print(hash*"#", end= "")
    print()

File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int h;
    do
    {
        h = get_int("Height: ");
    }

while (h>8 || h<1);

for (int i = 0; i < h; i++)

{

    for (int j=h-i; j>1; j--)

    {
        printf(" ");
    }

    for (int j = 0; j <= i; j++)

    {
        printf("#");
    }
    printf("  ");

    for (int j = 0; j<= i; j++)

    {
        printf("#");
    }

    printf("\n");

}

}
